src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -95,11 +102,37 @@ public void parseCommand_removetag() throws Exception {
         RemoveTagCommand command = (RemoveTagCommand) parser.parseCommand(
                 RemoveTagCommand.COMMAND_WORD + " " + "all" + " " + tag);
         assertTrue(command instanceof RemoveTagCommand);
+        assertEquals(new RemoveTagCommand(new Tag(tag)), command);
 
 
         RemoveTagCommand command2 = (RemoveTagCommand) parser.parseCommand(
                 RemoveTagCommand.COMMAND_WORD + " " + "5" + " " + tag);
```

Comment:
It is good attempt separating the different arguments used in testing, but as an improvement to readability, I suggest using local variables to denote arguments tested.

Reply:
Few other places where this improvement can be implemented.
Besides using local variables to denote arguments of the command to be tested, you can consider abstracting out generation of `RemoveTagCommand` in this instance.